pkgs/os-specific/linux/apparmor/default.nix

```diff
@@ -26,10 +33,16 @@ let
   };
 
   apparmor-sources = fetchurl {
-    url = "https://launchpad.net/apparmor/${apparmor-series}/${apparmor-version}/+download/apparmor-${apparmor-version}.tar.gz";
-    sha256 = "13xshy7905d9q9n8d8i0jmdi9m36wr525g4wlsp8k21n7yvvh9j4";
+    url = "https://launchpad.net/apparmor/${apparmor-series}/${apparmor-series}.${apparmor-patchver}/+download/apparmor-${apparmor-version}.tar.gz";
+    sha256 = "096zbg3v7b51x7f1ly61mzd3iy9alad6sd4lam98j2d6v5ragbcg";
```

Comment:
Nitpick: Maybe the most idiomatic way here is to have `apparmor-version = "3.0.1"` instead of `-series` and `-patchver` and use `${lib.versions.majorMinor apparmor-version}` instead of `${apparmor-series}`?

Reply:
Thanks